File: Lonely Number/NumberWithoutPair.java
class Main
{
    public static int numberWithoutPair(int[] arr) {
        int n = 0;

        for (int number: arr)
            n ^= number;

        return n;
    }


    public static void main(String[] args) {
        int[] arr = {1,9,5,9,3,8,1,3,5};
        System.out.println("Only number without pair is " + numberWithoutPair(arr));
    }
}

File: Lonely Number/number_without_pair.py
def number_without_pair(arr):
    n = 0
    for i in arr:
        n ^= i
    return n


if __name__ == "__main__":
    arr = [1,5,3,2,5,1,6,6,2]
    print("Only number without pair is", number_without_pair(arr))